next-env.d.ts

```diff
@@ -1,5 +1,6 @@
 /// <reference types="next" />
 /// <reference types="next/image-types/global" />
+/// <reference types="next/navigation-types/compat/navigation" />
```

Comment:
`yarn dev` added this automatically once I updated `next.config.mjs`

Reply:
are you using both `yarn` and `npm`? Is one recommended over the other for next / sanity?

Reply:
Oh sry I meant npm. Switching context between work where we use yarn